support-models/src/main/scala/com/gu/support/acquisitions/AcquisitionData.scala

```diff
@@ -0,0 +1,70 @@
+package com.gu.support.acquisitions
+
+import com.gu.support.encoding.Codec
+import com.gu.support.encoding.Codec.deriveCodec
+import io.circe.generic.semiauto.{deriveDecoder, deriveEncoder}
+import io.circe.{Decoder, Encoder}
+
+case class AbTest(
+  name: String,
+  variant: String
+)
+
+case class QueryParameter(
```

Comment:
AbTest and QueryParameter were defined in both the thrift and scala acquisition models.
They're actually equivalent.
I've moved them into support-models because support-frontend needs them as well, and it shouldn't have to depend on the acquisition-events module which is a separate concern.